src/evaluate.jl

```diff
@@ -247,7 +246,7 @@ function _evaluate(a::TaylorN{T}, vals::NTuple, ::Val{false}) where {T<:Number}
 end
 
 function evaluate(a::TaylorN{T}, vals::NTuple{N,Taylor1{S}}) where
-        {T<:Number, S<:NumberNotSeries, N}
+        {T<:NumberNotSeries, S<:NumberNotSeries, N}
```

Comment:
I think `Number` is correct here, or am I missing something?

Reply:
Thanks, yes I think this should be `Number`